llm_benchmark/benchmark/vllm_benchmark/benchmark_serving.py

```diff
@@ -132,69 +134,151 @@ def sample_sharegpt_requests(
     return filtered_dataset
 
 
+def sample_random_positive_int(mean: int, stddev: int) -> int:
+    ret = -1
+    while ret <= 0:
+        ret = int(random.gauss(mean, stddev))
+    return ret
+
+
+def sample_requests(
+    dataset_path: str,
+    num_requests: int,
+    tokenizer: PreTrainedTokenizerBase,
+    fixed_output_len: Optional[int] = None,
+    is_hf: bool = True,
+    input_column: str = "input",
+    output_column: Optional[str] = None,
+    mean_input_len: Optional[int] = None,
+    stddev_input_len: Optional[int] = None
+) -> List[Tuple[str, int, int]]:
+    if mean_input_len is not None and mean_input_len < 4:
+        raise ValueError("input_length must be at least 4")
+
+    try:
+        if is_hf:
```

Comment:
instead of this arg based condition, I would recommend using os.path to check if the dataset path is a file or  datasets library to check if it is a valid hf dataset